rmldatatfbbox/rmldatatfbbox/core.py

```diff
@@ -5,77 +5,158 @@
 Core command group and commands for TF Bounding Box dataset plugin.
 """
 import click 
-
-### OPTIONS ###
-
+import sys
+import io
+import importlib
+import shutil
+from pathlib import Path
+import tensorflow as tf
+from rmldatatfbbox.utils.helpers import (construct_all, write_label_map, export_as_TFExample)
+from ravenml.utils.question import user_input, cli_spinner
+from ravenml.data.helpers import default_filter_and_load
+from ravenml.data.write_dataset import write_dataset, write_metadata
+from ravenml.data.options import pass_create
+from ravenml.data.interfaces import CreateInput, CreateOutput
 
 ### COMMANDS ###
-@click.command(help='TensorFlow Object Detection with bounding boxes.')
+@click.group(help='Dataset creation with bounding box info')
```

Comment:
This should specifically say TensorFlow, since the output dataset is only for use with TF